doc/source/guzzle_sphinx_theme/guzzle_sphinx_theme/layout.html

```diff
@@ -81,6 +81,21 @@
           {%- block sidebar1 %}{{ sidebar() }}{% endblock %}
           {%- block document %}
             <div class="body">
+                <div class="well">
+                    <p>
+                        <strong>Note:</strong>
+                        You are viewing the documentation for an older version of AWS CLI (version 1).
```

Comment:
Let's update this sentence to be. Specifically, adding the words `the` and `major` to it.

```
You are viewing the documentation for an older major version of the AWS CLI (version 1).
```